core/src/main/java/apoc/trigger/Trigger.java

```diff
@@ -112,6 +116,7 @@ public Stream<TriggerInfo> list() {
                 );
     }
 
+    @Admin
```

Comment:
Should apoc.trigger.list also have @Admin annotation?

Reply:
Well yes, it probably makes more sense for all procedures to be `@Admin` for consistency.
I added the `apoc.trigger.list` annotation.
I'll add `trigger.show` to this pr when the other pr will be merged, or vice versa.